crates/node/src/ingest/dataset_controller.rs

```diff
@@ -124,6 +124,11 @@ impl DatasetController {
         let _ = self.compaction_enabled_sender.send(yes);
     }
 
+    pub fn get_first_block_number(&self) -> BlockNumber {
+        // self.first_block_receiver.borrow().clone()
+        unimplemented!()
```

Comment:
@eldargab what is the easiest way to find out the number of the lowest block stored?

Reply:
`db.snapshot().get_first_chunk()`